components/survey/List/index.module.scss

```diff
@@ -0,0 +1,24 @@
+.slickDots {
+  li {
+    button::before,
+    button:focus::before {
+      opacity: 0.2;
+
+      color: white;
```

Comment:
to keep consistency with TailWind, should we use the [@apply](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply) keyword? :)

Reply:
Thanks for the idea, done in 86c458775e4bb47df1b72df50f6dd54aba632aa2